umap/static/umap/js/modules/help.js

```diff
@@ -191,7 +191,7 @@ export default class Help {
     const container = DomUtil.add('div')
     DomUtil.createTitle(container, translate('Help'))
     // Special dynamic case. Do we still think this dialog is usefull ?
-    if (entries === 'edit') {
+    if (entries[0] === 'edit') {
```

Comment:
That exception looks smelly, can't we have a dedicated `showHelp()`?

Reply:
Done! :)